sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -398,6 +399,14 @@ numericLiteral
     | integerLiteral
     ;
 
+intervalLiteral
+    : INTERVAL sign=(PLUS | MINUS)? stringLiteral from=intervalField (TO to=intervalField)?
```

Comment:
Is it okay to restrict this to `stringLiteral` or do we also have to support more generic expressions to be fully SQL standard complient?

Reply:
Do we want to be complient to SQL standard core or against the full SQL 99 standard ?